### PR TITLE
👁 Eyes 👁

### PR DESCRIPTION
**Pull request by whmii**
screen shot 2017-03-10 at 3 39 19 pm

screen shot 2017-03-10 at 3 39 45 pm

screen shot 2017-03-10 at 2 52 45 pm

screen shot 2017-03-10 at 3 31 09 pm

### CODE REVIEW

**Comment by whmii** [comment]
I deleted a few hound comments about line length since refactoring the initializer stuff is in a separate upcoming PR

File: views/partials/eyes/eyes_1.erb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<circle id="simple-r" fill="rgba(47, 57, 71, 1)" cx="113.8" cy="77" r="5.7"/>
```
Comment (brendastorer) [inline views/partials/eyes/eyes_1.erb:1]: You technically don't need to make this `rgba` since it's full opacity. They could just be `rgb`

**Comment by brendastorer** [comment]
All eye emojis are creepy. 👀  👁‍🗨  Ship it.

File: views/partials/eyes/eyes_1.erb (hunk 1 above)
Comment (whmii) [reply views/partials/eyes/eyes_1.erb:1]: oh, all the other values are rgba so i figured I would just go with the flow

**Pull request merged**